stubs/pyjks/jks/util.pyi

```diff
@@ -0,0 +1,66 @@
+from collections.abc import Iterable, Mapping
+from pathlib import Path
+from struct import Struct
+from typing import Any
+from typing_extensions import Final, Literal, Self, TypeAlias
+
+from .bks import BksKeyEntry
+from .jks import PrivateKeyEntry
+
+b8: Final[Struct]
+b4: Final[Struct]
+b2: Final[Struct]
+b1: Final[Struct]
+py23basestring: Final[tuple[type[str], type[str]]]
+RSA_ENCRYPTION_OID: Final[tuple[int, ...]]
+DSA_OID: Final[tuple[int, ...]]
+DSA_WITH_SHA1_OID: Final[tuple[int, ...]]
+
+_KeystoreType: TypeAlias = Literal["jks", "jceks", "bks", "uber"]
+_PemType: TypeAlias = Literal["CERTIFICATE", "PUBLIC KEY", "PRIVATE KEY", "RSA PRIVATE KEY"]
+
+class KeystoreException(Exception): ...
+class KeystoreSignatureException(KeystoreException): ...
+class DuplicateAliasException(KeystoreException): ...
+class NotYetDecryptedException(KeystoreException): ...
+class BadKeystoreFormatException(KeystoreException): ...
+class BadDataLengthException(KeystoreException): ...
+class BadPaddingException(KeystoreException): ...
+class BadHashCheckException(KeystoreException): ...
+class DecryptionFailureException(KeystoreException): ...
+class UnsupportedKeystoreVersionException(KeystoreException): ...
+class UnexpectedJavaTypeException(KeystoreException): ...
+class UnexpectedAlgorithmException(KeystoreException): ...
+class UnexpectedKeyEncodingException(KeystoreException): ...
+class UnsupportedKeystoreTypeException(KeystoreException): ...
+class UnsupportedKeystoreEntryTypeException(KeystoreException): ...
+class UnsupportedKeyFormatException(KeystoreException): ...
+
+class AbstractKeystore:
+    store_type: _KeystoreType
+    entries: Mapping[str, AbstractKeystoreEntry]
+    def __init__(self, store_type: _KeystoreType, entries: dict[str, AbstractKeystoreEntry]) -> None: ...
+    @classmethod
+    def load(cls, filename: str | bytes | Path, store_password: str | None, try_decrypt_keys: bool = True) -> Self: ...
+    def save(self, filename: str | bytes | Path, store_password: str) -> None: ...
```

Comment:
I think you want `_typeshed.FileDescriptorOrPath` here, same as we use for the relevant overloads for `builtins.open`:

https://github.com/python/typeshed/blob/16b98846017c07ebbca9c75c099cb4b6c6b11ad8/stdlib/builtins.pyi#L1601-L1611

```suggestion
    def load(cls, filename: FileDescriptorOrPath, store_password: str | None, try_decrypt_keys: bool = True) -> Self: ...
    def save(self, filename: FileDescriptorOrPath, store_password: str) -> None: ...
```